ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommandTest.java

```diff
@@ -57,15 +61,19 @@ private String validCommandLine() {
         + " --signing-key-path="
         + PATH_TO_SIGNING_KEY
         + " --timeout="
-        + FIFTEEN;
+        + FIFTEEN
+        + " --tls-server-trust-store-file="
```

Comment:
same comment as before .. --tls-server-trust-store-file should be --hashicorp-tls-trust-store-file or similar??